database/tables/pl_courses.pg

```diff
@@ -1,4 +1,6 @@
 columns
+    banner_color: text
+    banner_html: text
```

Comment:
I didn't build editing for this into the admin UI; the expectation is that we'll set this in bulk with a manual query.

Reply:
Very reasonable.